lib/sift/client.rb

```diff
@@ -1,17 +1,22 @@
 require 'httparty'
 require 'multi_json'
 
+require_relative "./client/decision"
```

Comment:
1.8.7 is ancient — I think dropping compatibility is fine, but we'd want to update the readme as well